pillar.example

```diff
@@ -48,21 +48,22 @@ shorewall:
       comment: Reject all other connections
 
   rules:
-    - action: SSH(ACCEPT)
-      source: all
-      dest: all
-      comment: Allow SSH connections
-      ipv: 4                          # Rule for ipv4
-    - action: Ping(ACCEPT)
-      source: all
-      dest: $FW
-      comment: Allow Pings
-      ipv: 6                          # Rule for ipv6
-    - action: SaltMaster(ACCEPT)
-      source: all
-      dest: $FW
-      comment: Allow connections to saltmaster
-      # No ipv is given here          # Rule for ipv4 and ipv6
+    - NEW:      # available sections 'ALL', 'ESTABLISHED', 'RELATED', 'NEW' for all versions and additionally 'INVALID', 'UNTRACKED' for >= 4.6
```

Comment:
This test pillar snippet indicates that a dict is required rather than a list (i.e. `ALL:` vs. `- ALL:`):

https://github.com/saltstack-formulas/shorewall-formula/blob/086381d00f9417fe15cf8eaf7e399d2e783a6edc/.kitchen.yml#L57-L67

This line confirms how the pillar key is accessed (e.g. `salt['pillar.get']('shorewall:rules:NEW, [])`):

https://github.com/saltstack-formulas/shorewall-formula/blob/086381d00f9417fe15cf8eaf7e399d2e783a6edc/shorewall/files/rules.jinja#L28

---

The following suggestion proposes how this PR should be adjusted.  The comment has also been separated and spaced, in a way that would satisfy `yamllint` (which is used across many formulas in this org):

```suggestion
    # available sections 'ALL', 'ESTABLISHED', 'RELATED', 'NEW' for all versions and
    # additionally 'INVALID', 'UNTRACKED' for >= 4.6
    NEW:
```

Reply:
done - funny that it seems to work with both a list and a dictionary though, I'm not overly familiar with YAML, is that normal?

Reply:
@carinedengler I'm sure that the YAML rendering would be different but this is probably a feature of `salt['pillar.get']`, allowing a key to be extracted with either representation.

In any case, we'd prefer to keep the code consistent between `pillar.example` and the test pillar in `.kitchen.yml`.  Thanks for making the change.